src/App.tsx

```diff
@@ -1,73 +1,103 @@
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
 
-export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+interface AppProps {
+  debounceDelay?: number;
+}
+
+export const App: React.FC<AppProps> = ({ debounceDelay = 300 }) => {
+  const people: Person[] = useMemo(
+    () => [...peopleFromServer],
+    [peopleFromServer],
```

Comment:
you shouldn't use useMemo here cause 'peopleFromServer' aren't valid dependencies because mutating them doesn't re-render the component